powerplantmatching/package_data/config.yaml

```diff
@@ -23,11 +23,13 @@ matching_sources:
   - BEYONDCOAL
   - WIKIPEDIA
   - GGPT
+  - GCPT
 
 fully_included_sources:
   # Make individual queries for the datasets
   - BEYONDCOAL
   - GGPT
+  - GCPT
   - OPSD: Country not in ['Switzerland', 'Italy', 'Spain', 'Norway', 'Austria']
   - JRC: Country not in ['Switzerland', 'Albania', 'United Kingdom', 'Norway']
```

Comment:
I think you need to add the other sources as well to be tested. This contributing section described tests:
https://powerplantmatching.readthedocs.io/en/latest/contributing.html